sample-apps/react-native/dogfood/src/components/ParticipantsInfoList.tsx

```diff
@@ -65,12 +66,24 @@ export const ParticipantsInfoList = ({
     StreamVideoParticipant | undefined
   >(undefined);
   const call = useCall();
+
+  const environment = useAppGlobalStoreValue((store) => store.appEnvironment);
+
   const inviteHandler = async () => {
+    if (!call) {
+      throw new Error('Call not found');
+    }
     try {
+      const url =
+        environment === 'pronto'
+          ? `https://pronto.getstream.io/join/${call.id}`
+          : environment === 'pronto-staging'
+            ? `https://pronto-staging.getstream.io/join/${call.id}`
+            : `https://getstream.io/video/demos/join/${call.id}`;
       await Share.share({
-        url: `https://stream-calls-dogfood.vercel.app/join/${call?.id}`,
+        url,
         title: 'Stream Calls | Join Call',
-        message: `Join me on the call using this link https://stream-calls-dogfood.vercel.app/join/${call?.id}`,
+        message: `Join me on the call using this link ${url}`,
```

Comment:
For deeplinks, more additionals are needed unforunately, for example in AndroidManifest

Reply:
but this is a start. Sharing alteast works

Reply:
that's another story... we should fix that one as well. Now, this makes it easy to join on Web when started in RN.